00-intro/intro.md

```diff
@@ -15,39 +15,33 @@ links-as-notes: true
 
 ## Overview
 
-\Gls{NeoFS} is a decentralized distributed object storage system integrated with
-the [Neo Blockchain](https://neo.org).
+\Gls{NeoFS} is a decentralized distributed object storage system integrated with [Neo Blockchain](https://neo.org).
 
-\Glspl{Node} are organized in peer-to-peer network that takes care of storing
-and distributing user's data. Any Neo user may participate in the network and
-get paid for providing storage resources to other users or store his data in
-NeoFS and pay a competitive price for it.
+We store and distribute users' data across a peer-to-peer network of \Glspl{Node}. Whether a business or an individual, any Neo user may join the network and get paid for providing storage resources to others or pay a competitive price and enjoy NeoFS storage solution.
```

Comment:
The `enjoy` word doesn't provide any information here. Let's omit it.

Reply:
"enjoy" here simply means that "one can use", no personal attitude